src/app/log-tool/export-modal/export-modal.component.html

```diff
@@ -0,0 +1,40 @@
+<div bsModal #exportModal="bs-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="exportModalLabel"
+    aria-hidden="true" [config]="{backdrop: 'static'}">
+    <div class="modal-dialog modal-lg export-modal">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h3>Export Data</h3>
+                <button class="close" aria-label="Close" (click)="closeExportData()">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <div class="modal-body pb-0">
```

Comment:
I don't think we need the buttons here if we also have the buttons in the footer. I'd say just use the button in the footer and put a help text type of message in the body